lib/syncthing/syncthing.go

```diff
@@ -376,6 +389,17 @@ func (a *App) run() {
 	close(a.stopped)
 }
 
+func (a *App) EventLogger() (events.Logger, error) {
```

Comment:
As above. We took the event logger as a parameter so I don't think it's our responsibility to provide it to anyone else.

Reply:
That's cruft from when the logger wasn't passed as argument into `App` - will remove.